src/util/src/tmempool.c

```diff
@@ -125,6 +126,5 @@ void taosMemPoolCleanUp(mpool_h handle) {
   pthread_mutex_destroy(&pool_p->mutex);
   if (pool_p->pool) free(pool_p->pool);
   if (pool_p->freeList) free(pool_p->freeList);
-  memset(&pool_p, 0, sizeof(pool_p));
```

Comment:
this line shall be kept, it is a good practice, since app may still access this piece of memory.

Reply:
revised according to comment, please check again.